lib/ex_plasma/transaction/signed.ex

```diff
@@ -0,0 +1,131 @@
+defmodule ExPlasma.Transaction.Signed do
+  @moduledoc """
+  Representation of a signed transaction.
+
+  NOTE: before you use this, make sure you shouldn't use `Transaction` or `Transaction.Recovered`
+  """
+
+  alias ExPlasma.Crypto
+  alias ExPlasma.Transaction
+  alias ExPlasma.Transaction.Protocol
+  alias ExPlasma.Transaction.Witness
+  alias ExPlasma.TypedData
+  alias ExPlasma.Utils.RlpDecoder
+
+  @type tx_bytes() :: binary()
+  @type decoding_error() :: :malformed_rlp | mapping_error()
+
+  @type mapping_error() ::
+          :malformed_transaction
+          | :malformed_witnesses
+          | atom()
```

Comment:
```suggestion
  @type mapping_error() :: :malformed_transaction | :malformed_witnesses | atom()
```
what is the plain atom()?

Reply:
It's the mapping errors that can be returned when mapping the raw transactions